packages/frontend/src/main/components/auth/LoginBlurb.vue

```diff
@@ -24,6 +24,37 @@
         </span>
       </v-card-text>
     </div>
+    <div class="px-4">
+      <v-divider></v-divider>
+      <v-row align="center" justify="center" class="py-4">
+        <v-col cols="6">
+          <h3 class="text-h6 font-weight-bold mt-0">This is the Legacy Web App</h3>
+          <p class="mb-0 mt-1 primary--text text--disabled mr-2">
+            A better and more powerful web app is replacing this.
+            <a
+              href="https://speckle.systems/blog/the-new-way-to-collaborate-in-aec/"
+              target="_blank"
+              class="primary--text text--disabled"
+            >
+              Learn more.
+            </a>
+          </p>
+        </v-col>
+
+        <v-col cols="6" class="d-flex justify-end">
+          <v-btn
+            href="https://app.speckle.systems/"
```

Comment:
@benjaminvo @andrewwallacespeckle  Do we want to show this for everyone on all deployments? Surely someone self-hosting their own deployment won't be able to just go to app.speckle.systems and keep using the same accounts and everything?

Reply:
Benjamin confirmed that the login screen should remain as it is here